openupgradelib/openupgrade.py

```diff
@@ -1879,11 +1879,49 @@ def get_model2table(model):
         "ir.actions.act_url": "ir_act_url",
         "ir.actions.server": "ir_act_server",
         "ir.actions.client": "ir_act_client",
-        "ir.actions.report.xml": "ir_act_report_xml",  # model < v11
-        "ir.actions.report": "ir_act_report_xml",  # model >= v11
-        "mailing.contact.subscription": "mailing_contact_list_rel",  # v13 < model < v17
-        "mail.mass_mailing.list_contact_rel": "mail_mass_mailing_contact_list_rel",  # model = v12
     }
+    if version_info[0] < 8:
+        model2table.update(
+            {
+                "ir.actions.wizard": "ir_act_wizard",
+            }
```

Comment:
No need of all that lines (anyway, remember to remove the trailing comma for this to be friendly with pre-commit):

```suggestion
        model2table["ir.actions.wizard"] = "ir_act_wizard"
```

The same for the rest.